### DISCUSSION TITLE
What is the reason for the page being flagged?

### DISCUSSION THREAD

**Discussion by soylakate** [community/community#49903] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi :)
Can you please help me understand why the page is being flagged https://prnt.sc/xw3pEBaz2iO_ , hidden from public and how can it be visible again? Thank you

**Comment by maahaMaya** [comment, marked as answer]
Why account get flagged?
i. Due to creating more gist’s in one day your account gets flagged.

ii. Harass, abuse, threaten, or incite violence towards any individual or group, including GitHub employees, officers, and agents, or other GitHub Users;

iii. Use of servers for any form of excessive automated bulk activity (for example, spamming), or relay any other form of unsolicited advertising or solicitation through GitHub’s servers, such as get-rich-quick schemes;

iv. Attempt to disrupt or tamper with GitHub’s servers in ways that could harm Github’s Website or Service, to place an undue burden on GitHub’s servers through automated means, or to access GitHub’s Service in ways that exceed your authorization (other than those authorized by the GitHub Bug Bounty program);

v. Impersonate any person or entity, including any of GitHub’s employees or representatives, including through false association with GitHub, or by fraudulently misrepresenting your identity or site’s purpose; or violate the privacy of any third party, such as by posting another person’s personal information without consent.

How to remove the flag from Account?
Send below concern to GitHub support at 

developer@githubsupport.com

Hello, Mr. or Ms.
I don’t understand what happened to my account.
Would you tell me what to do now?
Thank you very much for your support.
I don’t know why my account has been flagged.

**Reply by soylakate** [reply]
Thank you!

**Comment by AbdulazizDXT** [comment]
Unfortunately, my account has been flagged for malware, and I also have more than one account. In fact, I don't have one of them. I have useful projects, and this is my second new account. My first account 4 years ago is unfortunately missing.
